tests/models/test_utils.py

```diff
@@ -79,24 +85,28 @@ def test_select_inducing_points(self):
         self.assertEqual(
             len(
                 select_inducing_points(
+                    allocator=KMeansAllocator(),
                     inducing_size=inducing_size,
                     covar_module=model.covar_module,
                     X=model.train_inputs[0],
                     bounds=model.bounds,
-                    method="kmeans++",
                 )
             ),
             20,
         )
 
-        with self.assertRaises(AssertionError):
```

Comment:
Assess whether the select_inducing_points function should e used at all anymore and consider replacing this check with a commensurate check that looks for invalid inputs (possibly for AutoAllocators).